app/src/main/java/com/avans/rentmycar/repository/RideRepository.kt

```diff
@@ -1,11 +1,11 @@
 package com.avans.rentmycar.repository
 
 import com.avans.rentmycar.api.RideService
-import com.avans.rentmycar.model.request.UpdateRideRequest
 import com.avans.rentmycar.model.response.RideResponse
+import com.avans.rentmycar.room.Ride
 
 class RideRepository {
-    suspend fun updateRide(updatedUser: UpdateRideRequest, rideId: String): retrofit2.Response<RideResponse>? {
-        return RideService.getApi()?.putRide(updatedUser, url="/api/v1/ride/${rideId}")
+    suspend fun updateRide(updatedUser: Ride, rideId: Long): retrofit2.Response<RideResponse>? {
+        return RideService.getApi()?.putRide(updatedUser, url ="/api/v1/ride/${rideId}")
```

Comment:
Actually it would be better to move the endpoint to the Ride service, what you did do for several POST request for the user service. But you can leave it for now, since we are running out of time.

Reply:
good point! I'll start with unit test now..